.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,22 @@
+name: Pull Requests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm test
+      - run: npm run build
```

Comment:
Only thing that is not running in `npm run version` that does `yarn build` would be curious if you think we should run that too or not.